api/Element.json

```diff
@@ -6617,7 +6619,8 @@
             },
             "opera": [
               {
-                "version_added": "24"
+                "version_added": "24",
+                "notes": "From version 78 returns a promise instead of <code>undefined</code>. The behaviour reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```

Comment:
```suggestion
              "notes": "From version 78, returns a promise instead of <code>undefined</code>. The behavior reflects <a href='https://github.com/w3c/pointerlock/pull/49'>a proposed specification change</a>."
```